### PR TITLE
Feature/wa 530 token item repo

### PR DESCRIPTION
**Pull request by DmitryBespalov**

### CODE REVIEW

File: MultisigWallet/MultisigWalletImplementations/DBTokeListItemRepository.swift [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            let statusString = rs.string(at: 2),
+            let status = TokenListItem.TokenListItemStatus(rawValue: statusString),
+            let updatedString = rs.string(at: 4),
+            let updated = DateFormatter.networkDateFormatter.date(from: updatedString) else { return nil }
+        let sortingId = rs.int(at: 3)
+        return TokenListItem(token: token, status: status, sortingId: sortingId, updated: updated)
+    }
+
+    public func all() -> [TokenListItem] {
+        return try! db.execute(sql: SQL.all, resultMap: tokenListItemFromResultSet).compactMap { $0 }
```
Comment (sche) [inline MultisigWallet/MultisigWalletImplementations/DBTokeListItemRepository.swift:93]: should be sorted by token code.

File: MultisigWallet/MultisigWalletImplementations/DBTokeListItemRepository.swift [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    public func whitelisted() -> [TokenListItem] {
+        return try! db.execute(sql: SQL.find_by_status,
```
Comment (sche) [inline MultisigWallet/MultisigWalletImplementations/DBTokeListItemRepository.swift:97]: should be sorted by sorting id

File: MultisigWallet/MultisigWalletImplementations/TokenListItemRepositoryTests.swift [hunk 3]
```diff
@@ -69,3 +70,55 @@ class InMemoryTokenListItemRepositoryTests: XCTestCase {
...
+        do_test_Find_Remove_All_Save()
+    }
+
+    func test_whenSavingWhitelistedItem_thenAssignsProperSortingOrder() {
+        do_test_whenSavingWhitelistedItem_thenAssignsProperSortingOrder()
+    }
+
+}
+
+class DBTokenListItemRepositoryTests: BaseTokenListItemRepositoryTests {
```
Comment (sche) [inline MultisigWallet/MultisigWalletImplementations/TokenListItemRepositoryTests.swift:92]: Is it Integration Test?

File: MultisigWallet/MultisigWalletImplementations/DBTokeListItemRepository.swift (hunk 1 above)
Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletImplementations/DBTokeListItemRepository.swift:93]: ooh then another field needed (token code) everywhere and select for all should have "ORDER BY token_code;"

File: MultisigWallet/MultisigWalletImplementations/DBTokeListItemRepository.swift (hunk 2 above)
Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletImplementations/DBTokeListItemRepository.swift:97]: this is sorted in SQL, please check

File: MultisigWallet/MultisigWalletImplementations/TokenListItemRepositoryTests.swift (hunk 3 above)
Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletImplementations/TokenListItemRepositoryTests.swift:92]: yes

**Review by sche — APPROVED** [review]

**Pull request merged**